src/popup/pages/Send/index.js

```diff
@@ -906,9 +906,9 @@ class SendPage extends React.Component {
     if(!isNumber(inputAmount) || !new BigNumber(inputAmount).gt(0)){
       return 0
     }
-    const {amount ,shares} = this.props.nodeDetail
+    const {amount, shares} = this.props.nodeDetail
     let sharePerAmount = new BigNumber(shares).dividedBy(amount).toString()
-    let realShare = new BigNumber(inputAmount).multipliedBy(sharePerAmount).toFixed(4,1).toString()
+    let realShare = new BigNumber(inputAmount).multipliedBy(sharePerAmount).toFixed(cointypes.decimals, BigNumber.ROUND_DOWN)
```

Comment:
here's one place we might not have to use ROUND_DOWN

Reply:
+1